lib/src/components/NxTable/NxTableCell.tsx

```diff
@@ -39,30 +41,42 @@ const NxTableCell = function NxTableCell(props: NxTableCellProps) {
   });
 
   let maskedSort;
-  if (sortDir === 'asc') {
-    maskedSort = (
-      <>
-        <NxFontAwesomeIcon icon={faSortDown} />
-        <NxFontAwesomeIcon icon={faSortUp} />
-      </>
-    );
-  }
-  else if (sortDir === 'desc') {
-    maskedSort = (
-      <>
-        <NxFontAwesomeIcon icon={faSortUp} />
-        <NxFontAwesomeIcon icon={faSortDown} />
-      </>
-    );
-  }
-  else {
-    maskedSort = <NxFontAwesomeIcon icon={faSort} />;
+  let ariaSort: 'ascending' | 'descending' | 'none' | undefined;
+  let ariaLabel;
+  if (isSortable) {
+    const text = textContent(children);
+    if (sortDir === 'asc') {
+      ariaSort = 'ascending';
+      ariaLabel = `${text} ${ariaSort}`;
```

Comment:
I found a website that suggested the aria-label should contain text telling the user what will happen when they click the cell, but in my testing that didn't work well. Instead I'm just labelling how the column is sorted. (It turns out when you click the cell it announces the _next_ label which isn't what I expected initially.)